.github/workflows/download-pulumi-cron.yml

```diff
@@ -0,0 +1,141 @@
+name: Download Pulumi Cron
+"on":
+  schedule:
+    - cron: 0 8 * * *
```

Comment:
I actually think it's worthwhile to run this more frequently (say hourly) and be okay with the fact that it might occasionally fail during a release where the versions will mismatch.

Reply:
I was going to run this once a day first then crank it up when we know it's actually doing the correct thing